### PR TITLE
feat(Harvest): init cozy-harvest

### PR DESCRIPTION
**Pull request by sizvix**
First step to initialise cozy-harvest-lib

### CODE REVIEW

**Comment by sizvix** [comment]
[(𝐒) [APR] Création d'une librairie](https://trello.com/c/fBQjTy6O/539-%F0%9D%90%92-apr-cr%C3%A9ation-dune-librairie)

File: packages/cozy-harvest/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Harvest
+=====
+
+Library to render a presentation for datas & configurations konnectors
```
Comment (gregorylegarec) [inline packages/cozy-harvest/README.md]: I think we can be more explicit, what do you think about:
> Cozy-Harvest will bring the logic, modules and components used in harvest application.
> This first version is just an "Hello World" version aimed to initialize the lib and make it available in NPM.

(My english is far to be perfect so maybe I made some typo.)

File: packages/cozy-harvest/package.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "cozy-harvest",
+  "version": "0.1.0",
+  "description": "Harvest library for auto-apps in Cozy",
```
Comment (gregorylegarec) [inline packages/cozy-harvest/package.json]: Same here, I think we can provide a better description:

> Provides logic, modules and components for Cozy's harvest applications.

What do you think ?

File: packages/cozy-harvest/README.md (hunk 1 above)
Comment (sizvix) [reply packages/cozy-harvest/README.md]: Maybe one word about what is "harvest application" ?
Harvest application => generic presentation for collected data ?

Comment (gregorylegarec) [reply packages/cozy-harvest/README.md]: You're right, maybe :

> Cozy-Harvest will bring the logic, modules and components used in harvest application.
>
> Harvest applications will be apps which will allow to configure konnectors, setup authentification informations and check for synchronized data (And much more in the future).
> Each harvest application will be associated to only one konnector.
>
> This first version is just an "Hello World" version aimed to initialize the lib and make it available in NPM.

File: packages/cozy-harvest/src/Harvest.spec.jsx [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import Harvest from './Harvest'
+import renderer from 'react-test-renderer'
```
Comment (gregorylegarec) [inline packages/cozy-harvest/src/Harvest.spec.jsx]: We are using Enzyme for shallow rendering, see https://github.com/cozy/cozy-store/blob/master/test/components/appsLoading.spec.js#L6 for example

File: packages/cozy-harvest/package.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    "test": "jest --verbose --coverage",
+    "build": "babel src -d dist",
+    "prepublishOnly": "yarn build",
+    "watch": "yarn build --watch"
+  },
+  "devDependencies": {
+    "@babel/cli": "7.1.2",
+    "babel-preset-cozy-app": "^1.2.1",
+    "react": "16.6.1",
+    "react-final-form": "3.6.7",
```
Comment (gregorylegarec) [inline packages/cozy-harvest/package.json]: I could be wrong but it seems that `react-final-form` is not needed in this initial version.

File: packages/cozy-harvest/package.json [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    "react-test-renderer": "16.6.1",
```
Comment (gregorylegarec) [inline packages/cozy-harvest/package.json]: We are using `enzyme` to render components in our tests, see further comment

**Comment by gregorylegarec** [comment]
Only a few remarks :)

**Comment by gregorylegarec** [comment]
Thanks @sizvix ! 💯

**Pull request merged**